bridge/build.rs

```diff
@@ -0,0 +1,17 @@
+use std::process::Command;
+
+fn main() {
+    // rerun build script if bridge contract has changed.
+    // without this cargo doesn't since the bridge contract
+    // is outside the crate directories
+    println!("cargo:rerun-if-changed=../contracts/bridge.sol");
+    let exit_status = Command::new("solc")
```

Comment:
Could we make this optional? So if there is no `solc` installed, precompiled code is used.

Reply:
we could. but then we'd be back to having to commit compiled code with people forgetting to update it etc.